src/index.html

```diff
@@ -2,12 +2,22 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Promise basics</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
   </head>
   <body>
     <h1 class="logo">MA</h1>
```

Comment:
The 'logo' class does not follow the BEM naming convention. Consider renaming it to 'header__logo' or something similar.